src/games/strategy/engine/lobby/server/LobbyGameController.java

```diff
@@ -66,10 +66,10 @@ public void postGame(final GUID gameID, final GameDescription description) {
     synchronized (m_mutex) {
       m_allGames.put(gameID, description);
     }
-    m_broadcaster.gameAdded(gameID, description);
+    m_broadcaster.gameUpdated(gameID, description);
```

Comment:
This change is the only one I'm not so sure about. As this is starting to potentially change public calls around lobby functionality. I believe since the method already existed and properly handles the "game added" situation that it shouldn't cause any issues but will need @veqryn to review this.